plutus-core/exe/Main.hs

```diff
@@ -647,13 +687,14 @@ runEval (EvalOptions language inp ifmt evalMode printMode printtime) =
                   CK  -> PLC.unsafeEvaluateCk  PLC.defBuiltinsRuntime
                   CEK -> PLC.unsafeEvaluateCek PLC.defBuiltinsRuntime
             body = void . PLC.toTerm $ prog
-        () <-  Exn.evaluate $ rnf body
+            !_ = rnf body
```

Comment:
This seems to give more consistent results than `Exn.evaluate`.  Is it causing deeper evaluation?

Reply:
It really shouldn't. Even with `Exn.evaluate` removed the original line should be forcing the program fully.

Reply:
I tried again and maybe I was imagining it, or there was something else going on.  I'll take another look later just to be sure.

Reply:
Yeah, not sure if there's really any difference.  There do seem to be slight differences for very fast programs, but it could just be statistical noise